NStackSDK/NStackSDK/Classes/Translations/TranslationManager.swift

```diff
@@ -471,6 +496,10 @@ public class TranslationManager {
     
     /// The URL used to persist downloaded translations.
     var translationsFileUrl: URL? {
-        return fileManager.documentsDirectory?.appendingPathComponent("Translations.nstack")
+        #if !(os(watchOS))
```

Comment:
Why did this need to be added? Is the watchOS changes included in this PR? Ideally this should be in the different branch/PR.